nixos/modules/system/boot/tmp.nix

```diff
@@ -3,62 +3,60 @@
 with lib;
 
 let
-  cfg = config.boot;
+  cfg = config.boot.tmp;
 in
 {
-
-  ###### interface
+  imports = [
+    (mkRenamedOptionModule [ "boot" "cleanTmpDir" ] [ "boot" "tmp" "cleanOnBoot" ])
+    (mkRenamedOptionModule [ "boot" "tmpOnTmpfs" ] [ "boot" "tmp" "onTmpfs" ])
+    (mkRenamedOptionModule [ "boot" "tmpOnTmpfsSize" ] [ "boot" "tmp" "tmpfsSize" ])
+  ];
 
   options = {
-
-    boot.cleanTmpDir = mkOption {
+    boot.tmp.cleanOnBoot = mkOption {
       type = types.bool;
       default = false;
       description = lib.mdDoc ''
         Whether to delete all files in {file}`/tmp` during boot.
       '';
     };
 
-    boot.tmpOnTmpfs = mkOption {
+    boot.tmp.onTmpfs = mkOption {
```

Comment:
Suggestion: `boot.tmp.mountTmpfs`

Reply:
I forget to push that yesterday but I already renamed it to useTmpfs.